src/Vault.sol

```diff
@@ -78,7 +78,7 @@ contract Vault {
      * @dev Throws if called by any account other than an authorised adress.
      */
     modifier onlyAuthorized() {
-        require(allowed[msg.sender], "V: You are not authorized");
+        require(allowed[msg.sender] || msg.sender == owner, "V: You are not authorized");
```

Comment:
new branch to remove onlyAuthorized on the 'increaseMargin' view function

Reply:
Already working on it in my take-leverage branch, can remove it here